<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stats_display_window</class>
 <widget class="QWidget" name="stats_display_window">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CS:GO Statistics</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="username_label">
     <property name="font">
      <font>
       <family>Arial [Mono]</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>username</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="realname_label">
     <property name="text">
      <string>realname</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="avatar_label">
     <property name="text">
      <string>avatar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
